### PR TITLE
docs: update the local testing to reflect latest changes in the process

### PR DESCRIPTION
**Pull request by fharper**
It's not new information, it should have been done a while ago.

### CODE REVIEW

**Review by CristhianF7 — APPROVED** [review]
Thanks for adding this @fharper.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -25,15 +25,37 @@ A [.devcontainer](https://containers.dev/) configuration is provided to allow fo
...
+go run . civo create --gitops-template-url https://github.com/kubefirst/gitops-template --gitops-template-branch main
+```
+
+#### Kubefirst API
+
+If you need to use a specific branch or latest from `main` that wasn't released yet for the [kubefirst-api](https://github.com/kubefirst/kubefirst-api) repository, you will need to first run it locally as described in [its documentation](https://github.com/kubefirst/kubefirst-api#running-locally). You will also need to run the code from [console](https://github.com/kubefirst/console) repository, whether you need to use a specific version of the code or not, as we don't expose the API directly. To do so, follow the [instructions in its README](https://github.com/kubefirst/console#setup-instructions). Before running the CLI as mentionned "The CLI" section, you need to export a local variable:
 
 ```shell
-go run . k3d create --gitops-template-url https://github.com/kubefirst/gitops-template --gitops-template-branch main
+export K1_CONSOLE_REMOTE_URL="http://localhost:3000"
```
[the remark this answers is not in the document]
Comment (fharper) [reply CONTRIBUTING.md:49]: This is written in the API README no? I tried to point people to the respective place where the information is. Or should I also add something here?

[the remark this answers is not in the document]
Comment (CristhianF7) [reply CONTRIBUTING.md:49]: No, we don't have this documented yet. If you don't mind I can add that change to your branch or do it in a following PR

[the remark this answers is not in the document]
Comment (fharper) [reply CONTRIBUTING.md:49]: Isn't in this table? https://github.com/kubefirst/kubefirst-api#environment-variables

[the remark this answers is not in the document]
Comment (fharper) [reply CONTRIBUTING.md:49]: I'll merge like that, and you can update if needed :)

**Pull request merged**